pkg/operation/operation.go

```diff
@@ -329,6 +329,26 @@ func (o *Operation) ReportShootProgress(ctx context.Context, stats *flow.Stats)
 	o.Shoot.Info = newShoot
 }
 
+// CleanShootTaskError removes the error with taskID from the shoot status
```

Comment:
What if the error for the given task ID is stored in `status.lastError` and not in the `status.lastErrors[]` list?

Reply:
Currently status.LastError doesn't have TaskIDs added to it during reconciliation. 
I can configure the logic in CleanShootTaskError to also check the TaskID of status.LastError. 

But there could be a minor inconvenience if someone adds TaskID to a status.LastError which is comprised of multiple errors -> errors on steps that haven't yet completed could be cleared.

Reply:
OK, I see, thank you for the explanation!